.travis.yml

```diff
@@ -1,5 +1,4 @@
 rvm:
-  - 1.9.3
```

Comment:
Why are we dropping support for 1.9.3? People still use that, and we should probably make sure Paperclip works on it, shouldn't we?